### PR TITLE
Fix invitations for paper withdraw and paper desk rejection

### PR DESCRIPTION
**Pull request by mohituniyal**
Use separate var for mail recipients and submission readers.
Add conference id as readers of the submission.

### CODE REVIEW

File: openreview/conference/templates/desk_reject_process.py [hunk 1]
```diff
@@ -16,6 +16,9 @@ def process(client, note, invitation):
     if PAPER_AREA_CHAIRS_ID:
         committee.append(PAPER_AREA_CHAIRS_ID)
     committee.append(PROGRAM_CHAIRS_ID)
+    committee.append(CONFERENCE_ID)
```
Comment (melisabok) [inline openreview/conference/templates/desk_reject_process.py]: if program chairs are already in the committee you don't need to add the conference id. 

I think the ECCV invitation doesn't have this version.

File: openreview/conference/templates/withdraw_process.py [hunk 2]
```diff
@@ -16,6 +16,9 @@ def process(client, note, invitation):
     if PAPER_AREA_CHAIRS_ID:
         committee.append(PAPER_AREA_CHAIRS_ID)
     committee.append(PROGRAM_CHAIRS_ID)
+    committee.append(CONFERENCE_ID)
```
Comment (melisabok) [inline openreview/conference/templates/withdraw_process.py]: same here.

ECCV version only has authors in the committee

File: openreview/conference/templates/desk_reject_process.py (hunk 1 above)
Comment (mohituniyal) [reply openreview/conference/templates/desk_reject_process.py]: Right, the ECCV version is missing the reviewers and ACs. When this PR is merged, I can update the Desk_Reject and Withdraw invitations with this version.

Comment (melisabok) [reply openreview/conference/templates/desk_reject_process.py]: I don't think we need to merge this PR, we need to use the version of master where AC/Reviewers are readers of the withdrawn/desk rejected notes and they also get email notifications. 

We already hace assigned ACs and we are going to have assigned Reviewers. They should get email notifications.

Comment (mohituniyal) [reply openreview/conference/templates/desk_reject_process.py]: Agreed. I will close this PR since this change is not needed anymore.

**Pull request closed without merging**